hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/GrpcBackendResolver.java

```diff
@@ -25,19 +25,20 @@
     Protocol protocol = EnrichedSpanUtils.getProtocol(event);
 
     if (protocol == Protocol.PROTOCOL_GRPC) {
-      String backendURI = SpanAttributeUtils.getStringAttribute(event,
-          Constants.getRawSpanConstant(Grpc.GRPC_HOST_PORT));
+      Optional<String> backendURI = RpcSemanticConventionUtils
+          .getGrpcURI(event);
 
-      if (StringUtils.isEmpty(backendURI)) {
+      if (backendURI.isEmpty() || StringUtils.isEmpty(backendURI.get())) {
         LOGGER.debug(
             "Detected GRPC span, but unable to find the {} attribute as the URI. Span Event: {}",
             Constants.getRawSpanConstant(Grpc.GRPC_HOST_PORT), event);
```

Comment:
We will have to change the debug message. `Unable to find attributes to derive URI` or something?